### ISSUE TITLE
Display Techniques for Pre-recorded Audio

### ISSUE THREAD

**Issue by wareid** [w3c/publ-a11y#286]
I am working through the ONIX Techniques for displaying metadata to determine how they translate to our systems and what we receive from publishers and authors. I have some particular concerns and questions about the "Pre-recorded Audio" section of the metadata, as I believe it's not entirely reflective of how audiobook content is handled or produced in market. I'll try to explain the issues in order of how they appear in the doc. 

Reading this section overall, I'm unsure of the intention behind it. Pre-recorded Audio content (audiobooks, audio tracks for synchronised media overlays, audio/video content within an EPUB) seems well represented by most of the variables, but I do feel it is missing key information about the accessibility of the pre-recorded audio content. We explain the intention of "Pre-recorded audio" really well in the [Principles](https://w3c.github.io/publ-a11y/UX-Guide-Metadata/draft/principles/#pre-recorded-audio) document as: 
> Indicates the presence of pre-recorded audio and specifies if this audio is standalone (an audiobook), accompanies text (embedded audio and video clips), or represents an alternative to the text (synchronized text-audio playback).

I would expect a similar structure in the ONIX and EPUB techniques documents. I do believe the EPUB techniques document achieves this. 

In [Understanding the Variables](https://w3c.github.io/publ-a11y/UX-Guide-Metadata/draft/techniques/onix-metadata/#understanding-the-variables-1), the variable `all_content_audio` is based on ONIX code 39 of list 196 (Supplementary material to an audiobook is accessible). While I see we explain the reasoning for this in that section, I think this variable will be confusing for publishers and inaccurate to how audiobooks and their supplemental content are currently presented to the user. Most supplemental content in market is provided separately from the audiobook, some audiobook platforms (mine included) do not bundle them together, and most audiobook platforms do not even have the ability to present supplemental content alongside the audiobook content. I think it would be more accurate for this variable to reflect the intention of the ONIX metadata field (ex. something like `supplemental_content_accessible`).

In [Understanding the Variables](https://w3c.github.io/publ-a11y/UX-Guide-Metadata/draft/techniques/onix-metadata/#understanding-the-variables-1), the variable `non_textual_content_audio_in_video` includes the ONIX codes 06 (Video), 24 (Animated / interactive illustrations), and 25 (Partial performance – video) from list 81. While its very likely content of this kind will contain audio, it's not guaranteed and we don't seem to offer a way for the content creator to specify that it does not. The intention with this variable seems unclear. 

In [Instructions](https://w3c.github.io/publ-a11y/UX-Guide-Metadata/draft/techniques/onix-metadata/#instructions-1) step 1, it currently says:
> IF `all_content_audio` AND NOT `synchronised_pre_recorded_audio`: THEN display "Audio only".

I believe this should include `audiobook` content (identified by ONIX code 01 of list 81), as currently in market most commercial audiobook content is audio-only. So instead I would expect this: 
> IF `all_content_audio` AND NOT `synchronised_pre_recorded_audio`, OR `audiobook`: THEN display "Audio only".

Similarly, step 2 in instructions says: 
> ELSE IF `audiobook` OR `non_textual_content_audio` OR `non_textual_content_audio_in_video` AND NOT `all_content_pre_recorded`: THEN display "Complementary audio and text"

I don't believe `audiobook` belongs in this instruction, as most (if not all) audiobooks in market do not have text at all. If step 1 is changed, removing `audiobook` from this step clears this up.

[Instructions](https://w3c.github.io/publ-a11y/UX-Guide-Metadata/draft/techniques/onix-metadata/#instructions-1) also appears to be incomplete, missing instructions for how to handle variables like `non_textual_content_audio` and `non_textual_content_audio_in_video` on their own or in relation to EPUB/textual content. 

EDIT: I realized upon further reading that what might be missing or necessary is a clearer definition or segmentation of audiobook content vs EPUB content. They are two different formats, and while it is true that audiobooks are pre-recorded audio content, they follow different practices and workflows in the industry. Since ONIX does have metadata in list 196 to address audiobook accessibility, I do think it's worth us specifying that in these documents, but I do think we need to clarify audiobook vs EPUB.

**Comment by chrisONIX** [comment]
Small comment on this sentence: “While its very likely content of this kind will contain audio, it's not guaranteed and we don't seem to offer a way for the content creator to specify that it does not. The intention with this variable seems unclear.”

ONIX list 81 does attempt to provide for this - code 50 - Video without audio and code 24 - Animated / interactive illustrations - are both meant for moving images without audio necessary for understanding the content.

**Comment by chrisONIX** [comment]
I agree that it is a good idea to make clear the different approach for audiobook content vs EPUB content (vs other content?) - this will make the display techniques easier for data receivers

**Comment by wareid** [comment]
> ONIX list 81 does attempt to provide for this - code 50 - Video without audio and code 24 - Animated / interactive illustrations - are both meant for moving images without audio necessary for understanding the content.

Code 24 is in the list currently, so maybe it should not be?

**Comment by chrisONIX** [comment]
I propose removing code 24 from the pre-recorded audio section.
At EDItEUR we will add to the note of list 81 - code 24 to make it clear that it has no audio recording necessary for understanding

**Comment by wareid** [comment]
@gautierchomel I see the commit removing code 24 from list 81, but that wasn't the only comment from this issue. I don't have concrete proposals for my other comments but I think they warrant some attention, unless I've missed that they are addressed in other issues.

**Comment by gautierchomel** [comment]
@wareid, are there still points in this issue that need to be discussed?

**Comment by GeorgeKerscher** [comment]
We have had major revisions realeted to the problems discussed in this issue.  I think we have address the problems, but if @wareid  could make sure all is well, that would be great.

If there are problems that still exist, please file a new issue. A new issue with the specific problems would be much easier to resolve.
